README.md

```diff
@@ -1,28 +1,26 @@
+The entire project can be run by opening [`game/index.html`](game/index.html) in a browser. All of the 3rd party dependencies like Babylon are using a CDN so you do not need Node or a local developer environment.
 
-***
+## 🧙‍ New Open Sourcerers 
+If you're new to contributing you can get started by creating a new repository by clicking `Use this template` and then selecting `Create a new repository` to create a copy of this repository into your own GitHub account. This is where you can make your own changes!
 
-If you're new to contributing you can get started by forking this repository into your own GitHub account or clicking the edit (pencil ✏️) button next to the file you want to change. Make the changes in your own repository and test them, then submit a Pull Request and link the issue you are contributing towards.
+![The green button used to create a new repository from this template](https://user-images.githubusercontent.com/35617441/234130644-bb6295d7-0499-4c3b-9214-495956201b09.png)
 
-If you would like to submit a new feature or fix a bug that isn't listed consider [Submitting an Issue](https://github.com/open-source-bootcamp/CubeFlyer/issues/new/choose) first so we can track the change!
-
-If you're new to Babylon JS that's fine to! Here are some helpful links to help you get started and remember you can always test your changes in your own branch *or* in the Pull Request first. Babylon runs in your browser and this game should work on your Phone, Tablet or PC - you can make and test your changes from any of those devices! If something doesn't work you can enter the developer mode of your browser (F12 on Edge or Chrome) and explore the errors in the console. This developer mode also lets you insert breakpoints if you want to debug further.
+If you're new to Babylon JS that's fine too! Here are some helpful links to help you get started. Babylon runs in your browser and this game should work on your Phone, Tablet or PC - you can make and test your changes from any of those devices! If something doesn't work you can enter the developer mode of your browser (F12 on Edge or Chrome) and explore the errors in the console. This developer mode also lets you insert breakpoints if you want to debug further.
 
 * The [Babylon.JS Playground](https://playground.babylonjs.com/) will let you explore different community samples to learn how Babylon works!
 * The [Babylon.JS API](https://doc.babylonjs.com/typedoc/modules/BABYLON) has documentation that can help you understand the API.
 
 We've also provided a file that should support launching from VS Code that will allow you to integrate your debugging experience there as well as a [devcontainer](https://code.visualstudio.com/docs/remote/create-dev-container) you can use to develop in if you choose with some recommended extensions pre-installed.
```

Comment:
Maybe we should mention Codespaces here (instead of devcontainers?) Also, in order for Codespaces to work we need to either run them locally in VSCode (which kind of defeats the purpose in this workshop) or use an extension like Live Server to be able to launch the project in the web.

We can talk more about it later :)